deepinv/physics/forward.py

```diff
@@ -346,6 +346,16 @@ def A_adjoint(self, y):
 
         return self.A_adj(y)
 
+    def A_grad(self, diff, x_est):
+        r"""
+        Computes the gradient of the forward operator :math:`\nabla_x f(A(x))`. It can make the representation of the gradient of the L2 loss more general and compact.
+
+        :param torch.Tensor diff: the difference between estimated measurements and true measurements.
```

Comment:
the parameter `x_est` is missing here

the description for `diff` needs to be updated